OpenTabletDriver/Vendors/Wacom/WacomTouchReport.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Numerics;
+using OpenTabletDriver.Plugin.Tablet;
+using OpenTabletDriver.Plugin.Tablet.Touch;
+
+namespace OpenTabletDriver.Vendors.Wacom
+{
+    public struct WacomTouchReport : ITouchReport, IAuxReport
+    {
+        public WacomTouchReport(byte[] report)
+        {
+            Raw = report;
+            AuxButtons = prevAuxButtons ?? new bool[4] { false, false, false, false };
+            Touches = PrevTouches ?? new TouchPoint[maxPoints];
+            if (report[2] == 0x81)
+            {
+                ApplyTouchMask((ushort)(Raw[3] | (Raw[4] << 8)));
+                PrevTouches = (TouchPoint[])Touches.Clone();
+                return;
+            }
+
+            var nChunks = Raw[1];
+            for (var i = 0; i < nChunks; i++)
+            {
+                var offset = i << 3;
+                var touchID = Raw[2 + offset];
```

Comment:
Should probably clean up to
```suggestion
            for (var i = 0; i < Raw[1]; i++)
            {
                var offset = (i * 8) + 2;
                var touchID = Raw[offset];
```
So it'll be easier to see that the chunks start on third byte, and has a length of 8 bytes.